### PR TITLE
There may or may not be a readable ZEND_TRACE_OP_INFO()

### PR DESCRIPTION
**Pull request by bwoebi**
Maybe I don't understand the code well enough, but I don't think it should happen. We had crashes there though, so that should probably should mitigate them.

### CODE REVIEW

**Review by iamluc — APPROVED** [review]

File: zend_abstract_interface/jit_utils/jit_blacklist.c [hunk 1]
```diff
@@ -203,6 +208,14 @@ void zai_jit_blacklist_function_inlining(zend_op_array *op_array) {
 
     size_t offset = jit_extension->offset;
 
+    // check whether the op_trace_info is actually readable or EFAULTing
+    // we can't trust opcache too much here...
+    char dummy_buf[sizeof(zend_op_trace_info)];
+    if (write(dd_probe_pipes[1], ZEND_OP_TRACE_INFO(opline, offset), sizeof(zend_op_trace_info)) < 0) {
+        return;
+    }
+    read(dd_probe_pipes[0], dummy_buf, sizeof(zend_op_trace_info));
```
[the remark this answers is not in the document]
Comment (bwoebi) [reply zend_abstract_interface/jit_utils/jit_blacklist.c:217]: I wish I could use it, but it requires glibc 2.19. Whereas we currently target 2.17.

[the remark this answers is not in the document]
Comment (morrisonlevi) [reply zend_abstract_interface/jit_utils/jit_blacklist.c:220]: Something, something, drop CentOS 7... :D

[the remark this answers is not in the document]
Comment (cataphract) [reply zend_abstract_interface/jit_utils/jit_blacklist.c:220]: You can use `syscall` directly and reference the syscall numbers and parameters [in this table](https://syscalls.mebeim.net/?table=x86/64/x64/latest).

[the remark this answers is not in the document]
Comment (bwoebi) [reply zend_abstract_interface/jit_utils/jit_blacklist.c:220]: @cataphract I suppose that will work, but using write() has the advantage of not being linux specific either.

**Pull request merged**